src/globus_sdk/_testing/data/globus_connect_server/get_collection_list.py

```diff
@@ -0,0 +1,70 @@
+import uuid
+
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+identity_id = str(uuid.uuid1())
+collection_ids = [str(uuid.uuid1()), str(uuid.uuid1())]
+gateway_ids = [str(uuid.uuid1()), str(uuid.uuid1())]
```

Comment:
`uuid.uuid1()` has serious collision problems. I recommend always using `uuid.uuid4()` to help ensure that values are unique.

```pycon
>>> import uuid
>>> 
>>> print("\n".join(str(uuid.uuid1()) for _ in range(5)))
c35c7812-59ee-11ed-8f60-67bc0b767b36
c35c7813-59ee-11ed-8f60-67bc0b767b36
c35c7814-59ee-11ed-8f60-67bc0b767b36
c35c7815-59ee-11ed-8f60-67bc0b767b36
c35c7816-59ee-11ed-8f60-67bc0b767b36
>>> 
>>> print("\n".join(str(uuid.uuid4()) for _ in range(5)))
00d6ceb0-0391-44dd-90aa-7fd9a1578299
f21a6479-5946-4b59-b82a-c1bda912fcf4
941b4f1b-f8e5-4646-a7d1-f2a07fb235b9
d07ee957-284d-4d83-aeb6-e4cd5bea847f
bde3dec8-f609-4c25-be17-217a042d5e0c
```

I'd be willing to bet that `collection_ids` and `gateway_ids` don't contain unique values when this runs.

```suggestion
identity_id = str(uuid.uuid4())
collection_ids = [str(uuid.uuid4()), str(uuid.uuid4())]
gateway_ids = [str(uuid.uuid4()), str(uuid.uuid4())]
```

Reply:
Ah, right, good catch! UUID1 is local timestamp based, and the resolution isn't good enough to guarantee uniqueness at this level.

I think of UUID4 as "slower", but it's so marginal that you need a very large number of calls to notice.

Just because I think it's interesting...
<details><summary>UUID v1 vs v4 vs v5 timing</summary>

This is what I ran, expecting v5 to be significantly faster. I was surprised to see that it's the slowest by far.
```
$ python -m timeit 'import uuid; namespace = uuid.uuid4(); x = [uuid.uuid5(namespace, "foo") for _ in range(100)]'
500 loops, best of 5: 496 usec per loop
$ python -m timeit 'import uuid; x = [uuid.uuid4() for _ in range(100)]'
500 loops, best of 5: 377 usec per loop
$ python -m timeit 'import uuid; x = [uuid.uuid1() for _ in range(100)]'
1000 loops, best of 5: 302 usec per loop
```

So our speedups are...

|      | v1      | v4      | v5    |
|-----|---------|---------|---------|
| v1 | 100% | 20%   | 39%  |
| v4 |  -        | 100% | 24%  |
| v5 |  -        | -         | 100% |


Of course, the scale here is microseconds! So it doesn't really matter. 😁 

</details>

Reply:
I wasn't aware of `uuid5()`, but it looks like v5 uses SHA-1 under the hood, which helps explain its "slowness". :grinning: